Zapper-fi/studio
# feat(mux): Integration MUX to Studio

## Description
Integration MUX to Studio


## Checklist


## How to test?
1. edit/create `.env` adding `ENABLED_APPS=mux`
2. run `pnpm dev` to spin up nest server
3. visit dev endpoints below to see result of tvl to integrate mux
* http://localhost:5001/apps/mux/tvl?network=binance-smart-chain
* http://localhost:5001/apps/mux/tvl?network=arbitrum
* http://localhost:5001/apps/mux/tvl?network=avalanche
* http://localhost:5001/apps/mux/tvl?network=fantom
